### PR TITLE
[zookeeper] adds zookeeper chart

### PR DESCRIPTION
**Pull request by mms-gianni**
### Description of the change

This PR adds a the Zookeeper Chart similar to [Bitnamis Zookeeper Chart](https://github.com/bitnami/charts/tree/main/bitnami/zookeeper) . It is fully capable to run as a Cluster (was quite a challenge get it running). The config has a metrics section, which is implemented as described in the docs, but I haven't tested it yet. Same with the Admin.

There is no ingress yet. Im not sure if this makes sense for zookeeper.

SCR-20250911-hqfk

### Benefits

This chart can be used as a subchart for other Helm charts like Kafka. 

### Possible drawbacks

None

### Applicable issues

None

### Additional information

- The Chart icon needs to be uploaded. 
- The Artefacthub RepositoryID hast to be replaced with actual ID

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md`
- [X] Title of the pull request follows this pattern [<name_of_the_chart>] Descriptive title

### CODE REVIEW

**Comment by zOnlyKroks** [comment]
Good Morning @mms-gianni,
thank you for the addition of this helm chart and the work you put into it. Quite frankly tho, I am uncertain if we can offer support / further maintenance, outside of bumping versions with the help of renovate for this chart, as we have never worked with zookeeper. If however, you are down to actively maintaining this chart / fixing issues that arise with this chart, we are more than happy to merge it. Thanks again.
LG,
Finn Rades

**Comment by mms-gianni** [comment]
Hi Finn

Lucky you haven't worked with it; I'm not a fan either. It's used in Kafka, Solr, and Elasticsearch.

Bumping version might be enough for this chart, since zookeeper is very stable. Specially the official image. So I don't exepect any new features or a lot of bugs.
 
I consider the missing Admin feature as an edge case and I can offer to implement it when someone requests it.

We plan to use this chart in production within the next 2-3 weeks. I'll test the metrics when I got a realworld workload using it.

**Comment by zOnlyKroks** [comment]
> Hi Finn
> 
> Lucky you haven't worked with it; I'm not a fan either. It's used in Kafka, Solr, and Elasticsearch.
> 
> Bumping version might be enough for this chart, since zookeeper is very stable. Specially the official image. So I don't exepect any new features or a lot of bugs.
> 
> I consider the missing Admin feature as an edge case and I can offer to implement it when someone requests it.
> 
> We plan to use this chart in production within the next 2-3 weeks. I'll test the metrics when I got a realworld workload using it.

Okay then, if okay, I would @ you in issues that arise on the issue tracker, as I have no clue on how zookeeper works and you could probably provide better help to users than I could. But anyways, I would merge this later today if your ok with that.

**Comment by mms-gianni** [comment]
Perfect. No hurry for merging.

**Comment by mms-gianni** [comment]
I can add my contacts in maintainers section if you want.

**Comment by zOnlyKroks** [comment]
> I can add my contacts in maintainers section if you want.

Would be great

**Comment by zOnlyKroks** [comment]
You could maybe also fix the linting issues while your at it, mentioned in the runner

**Comment by dloewen2** [comment]
Hi @mms-gianni, 
we already created a logo and artifact-repository. Could you add these to your Pull-Request:
Chart.yaml Icon URL: https://a.storyblok.com/f/143071/512x512/07b0f90b58/apache-zookeeper-logo.svg
Artifacthub-repo-id: 8f2d141f-1c72-4691-b52e-46e93382b358

Then we wouldn't need to update these after the PR is merged.

File: charts/zookeeper/values.yaml [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    admin: 8080
+
+## @section Resources
+resources:
+  {}
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  #limits:
```
Comment (zOnlyKroks) [inline charts/zookeeper/values.yaml]: missing space

File: charts/zookeeper/values.yaml [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  #  memory: 512Mi
+  #requests:
```
Comment (zOnlyKroks) [inline charts/zookeeper/values.yaml]: missing space again (lint fails because of this)

**Comment by zOnlyKroks** [comment]
Thank you for this
